### ISSUE TITLE
ImportError: cannot import name 'collections_abc' from 'six.moves' (unknown location) on label-studio init labeling_project

### ISSUE THREAD

**Issue by g-simmons** [heartexlabs/label-studio#447]
**Describe the bug**
Hi all, 
- I created a new conda environment (`conda create -n labelstudio`), 
- activated the environment (`conda activate labelstudio`), 
- installed label-studio (`pip install labelstudio`) 
- try to init a project (`label-studio init labeling_project`), 
- and received the following error. 

I thought maybe this was because I omitted specifying the python version when creating the conda env, so I did 
- `conda install python=3.8`, 
- re-ran `label-studio init labeling_project` and got the same error.

**FIX**: this was solved by `pip install -U six`

**To Reproduce**
Above

**Expected behavior**
No error on `label-studio init labeling_project`

**Screenshots**
```
Traceback (most recent call last):
  File "/Users/gabe/opt/miniconda3/envs/labelstudio/bin/label-studio", line 5, in <module>
    from label_studio.server import main
  File "/Users/gabe/opt/miniconda3/envs/labelstudio/lib/python3.8/site-packages/label_studio/server.py", line 50, in <module>
    from label_studio.utils.uri_resolver import resolve_task_data_uri
  File "/Users/gabe/opt/miniconda3/envs/labelstudio/lib/python3.8/site-packages/label_studio/utils/uri_resolver.py", line 8, in <module>
    from google.cloud import storage as gs
  File "/Users/gabe/opt/miniconda3/envs/labelstudio/lib/python3.8/site-packages/google/cloud/storage/__init__.py", line 38, in <module>
    from google.cloud.storage.batch import Batch
  File "/Users/gabe/opt/miniconda3/envs/labelstudio/lib/python3.8/site-packages/google/cloud/storage/batch.py", line 31, in <module>
    from google.cloud.storage._http import Connection
  File "/Users/gabe/opt/miniconda3/envs/labelstudio/lib/python3.8/site-packages/google/cloud/storage/_http.py", line 17, in <module>
    from google.cloud import _http
  File "/Users/gabe/opt/miniconda3/envs/labelstudio/lib/python3.8/site-packages/google/cloud/_http.py", line 22, in <module>
    from six.moves import collections_abc
ImportError: cannot import name 'collections_abc' from 'six.moves' (unknown location)
```

**Environment (please complete the following information):**
 - OS: OSX Catalina
 - Browser haven't opened the browser yet :) 
 - Version: Label Studio version: 0.7.4.post0

**Comment by makseq** [comment]
@g-simmons It's known issue, we fixed it in release/0.7.5. 
Or you can use 0.7.4.post1:
 ```
pip install label-studio==0.7.4.post1
```

**Comment by g-simmons** [comment]
Oh ok no worries - already resolved, feel free to close, just wanted to make sure you were aware. Loving the tool so far 👍 👍
